### ISSUE TITLE
Should we eat Canned Foods?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#77]
While researching the various types of Processed Foods: https://github.com/dwyl/health/issues/78
Is there any scientific evidence that eating canned foods e.g: soaked beans or chopped tomatoes is _negative_ to our health...?

I _suspect_ that data will be difficult to come by for this because there is an entire _industry_ reliant on selling canned foods ... 

This WebMd article by Alia Hoyt suggests we should _avoid_ Canned foods: 
http://www.webmd.com/food-recipes/features/can-the-canned-food
Note: Alia is a _journalist_ https://www.linkedin.com/in/alia-hoyt-8b851968 and does not _appear_ to have any medical training _however_ most of the article is quoted from Wendy Palmer  (_MS, RD, LD, CHES, Program Manager, Child Wellness, Children’s Healthcare of Atlanta ..._) 
The conclusion is that most canned foods contain **too much _Sodium_** (_Chloride i.e. salt_) and should therefore be avoided as too much salt leads to Hypertension.

Found this (_admittedly not very "scientific" article_) stating that people who eat canned foods 
http://www.dailymail.co.uk/health/article-3362515/Tinned-food-good-People-eat-canned-fruit-veg-healthier-diets-eat-none-all.html

> My _preliminary_ finding on this is that we should consider _frozen_ foods over canned wherever possible. I'm going to buy _dry_ beans for my soup from now on even though they require more time/effort.

**Comment by nelsonic** [comment]
Recently re-read ["How Not to Diet"](https://nutritionfacts.org/book/how-not-to-diet/) by Dr. Michael Greger and was reminded of the fact that the lining of cans is **`BPA`** which is considered an endocrine disruptor (ED) and estrogenic chemical. https://en.wikipedia.org/wiki/Bisphenol_A#Health_effects

So I'm going to phase out canned foods as much as possible, which is lame for Coconut Milk ... 🥥  💭

**Comment by carolrmc** [comment]
Coconut milk is available in bottles too, but the quality is not as good!
It’s the convenience of cans that makes it hard to phase them out -
sweetcorn for instance doesn’t come in a glass jar. (And so many other
products that are marketed in plastic jars)

**Comment by iteles** [comment]
Actually, we can make coconut milk at home pretty trivially!

Either sweetened or unsweetened, based on this recipe: https://minimalistbaker.com/how-to-make-coconut-milk/

It's basically shredded coconut and water. 

And we can get shredded coconut from Transagri at €5,70 a kilo: https://transagri-lda.com/produto/coco-ralado

**Comment by carolrmc** [comment]
True, but like all things, if it’s available commercially it’s more
convenient! And less time consuming for the busy life we have😔
